pkg/cmd/codespace/ssh.go

```diff
@@ -147,6 +155,129 @@ func (a *App) SSH(ctx context.Context, sshArgs []string, opts sshOptions) (err e
 	}
 }
 
+func (a *App) ListOpensshConfig(ctx context.Context, opts configOptions) error {
```

Comment:
Does this method need to be public? If so, it should have a doc comment.

capitalization: ListOpenSSHConfig

Better name: PrintOpenSSHConfig ?

Reply:
Sure, that's a better name, I'll change it. I created this by adapting [List](https://github.com/cli/cli/blob/0d3dd7e75851342f97dd1758cabe187e946dd634/pkg/cmd/codespace/list.go#L37), which is also why my implementation was public.